buildroot-external/overlay/base/etc/lighttpd/conf.d/sslsettings.conf

```diff
@@ -1,9 +1,6 @@
+server.port = 443
```

Comment:
Is this recent addition really right? because we use ssl not only for port 443 but also for all the 4xxxx ports in `proxy_normal.conf`.

Reply:
Yes, added.  If enabling `ssl.engine = "enable"` in global scope, I wanted it to apply to port 443, not the default (`server.port = 80`).  All the `$SERVER["socket"]` which set `ssl.engine = "enable"` will inherit the global TLS config if they do not set their own `ssl.*` directives besides `ssl.engine = "enable"`.